inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -472,6 +472,18 @@ std::string get_output_precision_name(ngraph::Output<Node>& o) {
     }
 }
 
+std::string escape_delim(const std::string& name) {
+    std::string result_name = name;
+    size_t index(0);
+    while (index != std::string::npos) {
+        index = result_name.find(',', index);
+        if (index == std::string::npos) break;
+        result_name.replace(index, 1, "\\,");
+        index += 2;
+    }
+    return result_name;
+}
```

Comment:
The result of `index += 2;` could be greater than `std::string::npos`. Also this code can be simplified:
```suggestion
std::string escape_delim(std::string name)
{
    size_t index = name.find(',', 0);
    while (index != std::string::npos) {
        name.replace(index, 1, "\\,");
        index = name.find(',', index + 2);
    }
    return name;
}
```

Reply:
The result of `index += 2;` cannot be greater than `std::string::npos` because I add 2 symbols before this increment.
In any case I agree that your proposal looks simplier.
Fixed